src/main/app/src/constants.ts

```diff
@@ -142,7 +142,7 @@ export const HAKULOMAKETYYPPI = {
   HAKUAPP: 'haku-app',
   MUU: 'muu',
   EI_SAHKOISTA_HAKUA: 'ei sähköistä',
-};
+} as Record<any, HakulomakeTyyppi>;
```

Comment:
any varmaan voisi olla string

Reply:
Mietin että voisiko koko common.d.ts:ssä olevan HakulomakeTyyppi union typen poistaa ja tehdä tästä vaan enum:in jolla korvata commonin HakulomakeTyypin käyttöpaikat?

Reply:
@vesaruu: Tuon voi tehdä, mutta joudun siirtämään HakuFormValues-tyypin tavalliseen `.ts`-tiedostoon. Jos `.d.ts`-tiedostossa importoi tavallisen `.ts`-tiedoston, sen tyypit eivät ole enää globaalisti käytettävissä. HAKULOMAKETYYPPI täytyy myös olla tavallisessa `.ts`-tiedostossa enuminakin, että sitä voi käyttää muualla sovelluksessa. Varmaan se näin onkin parempi, vaikka noita tyyppejä joutuu sitten importoimaan.